pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -157,6 +165,7 @@ func (i *Gatherer) Gather(ctx context.Context, recorder record.Interface) error
 		GatherCRD(i),
 		GatherHostSubnet(i),
 		GatherMachineSet(i),
+		GatherStatefulSets(i),
```

Comment:
Why are we gathering this?  It's not clear to me what the value is here

Reply:
I'm going to skip this this week, until we know it has value.

Reply:
We are gathering statefulsets only from `openshift-*` namespaces. There was the nomination https://issues.redhat.com/browse/INSIGHTOCP-50 and corresponding ccx issue https://issues.redhat.com/browse/CCXDEV-2338

Reply:
If I understand the linked issues, the request is to determine if the cluster has configured persistent storage for monitoring. That can be determined from the cluster monitoring configuration.
https://docs.openshift.com/container-platform/4.6/monitoring/configuring-the-monitoring-stack.html#configuring-a-local-persistent-volume-claim_configuring-the-monitoring-stack

```
oc -n openshift-monitoring get configmap cluster-monitoring-config
```

Reply:
Ahh OK thanks. I didn't know that. Yeah this would make more sense. I'll try to reimplement it.